installers/ansible/roles/pgo-operator/defaults/main.yml

```diff
@@ -40,6 +40,7 @@ pod_anti_affinity_pgbackrest: ""
 pod_anti_affinity_pgbouncer: ""
 
 namespace: ""
+namespace_mode: "dynamic"
```

Comment:
Noting for the defaults between `dynamic` and `disabled`

Reply:
Sticking with default of `dynamic` per discussion.